### PR TITLE
Fix space around tuples

### PR DESCRIPTION
**Pull request by gpetiot**
From https://github.com/janestreet/ocamlformat/commit/4feebdd2e42acf7c64c3eb58eda606ca52548b6b

### CODE REVIEW

File: src/Fmt_ast.ml [hunk 1]
```diff
@@ -352,6 +352,16 @@ let wrap_record c =
   if c.conf.space_around_collection_expressions then wrap "{ " "@ }"
   else wrap_fits_breaks c.conf "{" "}"
 
+let wrap_tuple ~parens ~no_parens_if_break c =
+  let wrap_fits_breaks_if _c cnd pre suf k fs =
```
Comment (hhugo) [inline src/Fmt_ast.ml]: This should probably move to [Fmt]

Comment (hhugo) [reply src/Fmt_ast.ml]: Also, I wonder why code for tuple is so different from list, array, record  ..

File: src/Fmt_ast.ml [hunk 2]
```diff
@@ -352,6 +352,16 @@ let wrap_record c =
...
+    fits_breaks_if cnd pre (pre ^ " ") fs ;
+    k fs ;
+    fits_breaks_if cnd suf ("@ " ^ suf) fs
+  in
+  if parens then wrap_fits_breaks_if c.conf true "(" ")"
```
Comment (Julow) [inline src/Fmt_ast.ml]: Why did you rewrite `wrap_fits_breaks_if` ? The one from `Fmt` should work fine here.

File: src/Fmt_ast.ml (hunk 1 above)
Comment (gpetiot) [reply src/Fmt_ast.ml]: I'm not sure this function would fit in Fmt, for now Fmt only contains generic functions that are not designed for any particular AST node.
If we want to extract wrap_tuple, wrap_record and so on, maybe we should create a file Fmt_ast_helper to gather the AST specific functions.

**Pull request merged**